### ISSUE TITLE
[9.3.1] Computer -> add -> locked "Data center position"

### ISSUE THREAD

**Issue by Rico29** [glpi-project/glpi#4646]
* Steps to reproduce (which actions have you made) :
Add elements in "Management > Data centers"
Add a computer in "computers"

* Expected result :
Ability to choose "Data center position"

* Actual result :
"Data center position" is locked to one of the Data Centers

* URL of the page :
https://glpi.celya.fr/front/computer.form.php?id=-1&withtemplate=2

* Screenshot of the problem (if pertinent) :
![selection_027](https://user-images.githubusercontent.com/12935571/45616449-cee43a80-ba6f-11e8-8aab-a9339227efbe.png)

* Your GLPI setup (you can find it in Setup > General menu, System tab) :
```
GLPI 9.3.1-git-9.3/bugfixes-7b251c44a ( => /usr/share/glpi_9.3)
Installation mode: GIT
```

Regards
Cédric

**Comment by trasher** [comment]
See https://github.com/glpi-project/glpi/issues/4648

**Comment by Mirkk** [comment]
Dear all, this issue seems not at all related to #4648 and should thus not be closed!

As @Rico29 describes, the datacenter position is a fixed string (still in 9.4, 9.5 not tested.) and not editable from the computer view (like the "Location"). Moreover, it is not available when the computer is not associated to a rack, so you are unable to associate a computer to a rack from the computer view. All this was possible with the old plugin, because there was a menu entry available on the left side.
This is particularly annoying in huge datacenters where daily new computers need to be associated to racks.

This should help to interpret @Rico29s expected result : Ability to choose "Data center position"

Thanks!
